exercises/en/test_02_20b.py

```diff
@@ -0,0 +1,13 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+    assert set(bs_column.values) == {'strong', 'weak'}, "Have you selected the 'base_score' column?"
+    assert list(score_freq) == [573, 228], "Count values are incorrect. Are you using the 'count' function?"
```

Comment:
I generally like it better when you have "Your" or "The" in front of the statement. "Count values are incorrect." is a bit abrupt. Can you make these changes globally please?

Reply:
Maybe sort the list first?